apps/dot-voting/app/components/VotingCardGroup.js

```diff
@@ -58,4 +63,26 @@ VotingCardGroup.propTypes = {
   children: PropTypes.node.isRequired,
 }
 
+const DiscTag = styled.span`
+  display: inline-flex;
+  white-space: nowrap;
+  color: rgb(109, 128, 136);
+  padding-top: 4px;
+  letter-spacing: -0.5px;
+  /* stylelint-disable-next-line property-no-vendor-prefix */
+  -webkit-box-pack: center;
+  justify-content: center;
+  /* stylelint-disable-next-line property-no-vendor-prefix */
+  -webkit-box-align: center;
```

Comment:
Do we need these two stylelint-ignored lines? Are they not auto-inserted for `justify-content: center`?

Reply:
At least in my browser all of that CSS was required and both ignores needed to be present, as they are very specific and have very specific targets.